src/renderer/components/ChatView.js

```diff
@@ -204,12 +204,16 @@ class RenderMessage extends React.Component {
     }
 
     if (msg.file) {
-      props.attachment = { url: msg.file, contentType: convertContentType(message.filemime), filename: msg.text }
+      props.attachment = {
+        url: msg.file,
+        contentType: convertContentType(message.filemime),
+        filename: msg.text
+      }
     } else {
       props.text = msg.text
     }
 
-    return (<Message {...props} />)
+    return (<div className='MessageWrapper'><Message {...props} /></div>)
```

Comment:
this would also be a good case for `styled-components`

Reply:
TIL